### ISSUE TITLE
How's this work with React Native for Android?

### ISSUE THREAD

**Issue by elie222** [hharnisc/react-native-meteor-websocket-polyfill#6]
Would it be a problem?

**Comment by devlesedi** [comment]
I second this.

**Comment by Sh3kar** [comment]
I tried to connect react-native + Android + DDP, but I'm getting Error: process.nextTick is not a function

**Comment by tgoldenberg** [comment]
Yes, this would be awesome if it could work with RN Android. Any ideas on what's blocking it?

**Comment by jmurth1234** [comment]
Would also like to know if this will work with android - anyone had any luck?

**Comment by tgoldenberg** [comment]
[This repo](https://github.com/spencercarli/meteor-todos-react-native-2) seems to work just fine @rymate1234
